RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java

```diff
@@ -36,28 +36,23 @@ class DefaultTokenFactory implements TokenFactory {
 	private int currentFreeToken;
 
 	protected static final int DEFAULT_START_SIZE	= 30;
-	protected static final int DEFAULT_INCREMENT		= 10;
-
 
 	/**
 	 * Constructor.
 	 */
 	DefaultTokenFactory() {
-		this(DEFAULT_START_SIZE, DEFAULT_INCREMENT);
+		this(DEFAULT_START_SIZE);
 	}
 
 
 	/**
 	 * Constructor.
 	 *
 	 * @param size The initial number of tokens in this factory.
-	 * @param increment How many tokens to increment by when the stack gets
-	 *        empty.
 	 */
-	DefaultTokenFactory(int size, int increment) {
+	DefaultTokenFactory(int size) {
```

Comment:
The change should be tagged with the JIRA

Reply:
Done now.